package.json

```diff
@@ -19,6 +19,7 @@
   "peerDependencies": {
     "@angular/core": ">=2.0.0",
     "@angular/platform-browser-dynamic": ">=2.0.0",
+    "core-js": "^3.0.0",
```

Comment:
This would be a breaking change, while this preset works with `core-js@2`. Can we make it `"core-js": ">=2.0.0 < 4.0.0",` for now and adjust the `setupJest` to try imports from `core-js` v2 and v3?

Reply:
I am thinking to `try/catch` the `require()` functions. Will that be a good idea?

Reply:
If we want to support both, this is the only solution I can see for now :D

Reply:
I got a small issue :)  
How am I supposed to test the changes? I can't make tests fail.

I deleted `./setupJest.js`, ran `npm run prepare`, and then both `npm run test` are green in `./` and `./example`

Reply:
We'd need to have 2 projects, which is slow to test unfortunately. What you can  do is to import `./setupJest.js` from a new test file and mock `require` there to behave appropriately for v2 and v3

Reply:
I think a `require`-mock is appropriate. We do not have to test node being able to resolve and load an installed package, we just have to test if the errors are handled. So mocking require and throwing the corresponding error as node would do should suffice in my opinion.

If no `core-js` installation can be found, an appropriate error should still be thrown.